app/src/app/services/connectors/rest-api/rest-api-connector.service.ts

```diff
@@ -102,7 +102,10 @@ export class RestApiConnectorService extends ApiConnector {
      */
     private getObjectName(object: StixObject): string {
         if (object.type == "relationship") {
-            return `${object["source_name"]} ${object["relationship_type"]} ${object["target_name"]}`
+            if (object["source_name"] != '[unknown object]' && object["target_name"] != '[unknown object]') {
```

Comment:
Could you switch the conditionals here like this, and then switch the return statements to match? Generally I find it easier when coming across an `if` statement to deal with it in the positive sense as opposed to the negative. That said, looks good after that!

```suggestion
            if (object["source_name"] == '[unknown object]' && object["target_name"] == '[unknown object]') {
```